src/main.rs

```diff
@@ -208,6 +220,30 @@ impl Op {
             }
         }
 
+        Ok(())
+    }
+    fn print_changelog(&self, resolver: &Resolver) -> Result<(), Error> {
+        match self {
+            Op::Update(old, new) => {
+                let old = resolver.pkg(old)?;
+                let new = resolver.pkg(new)?;
+                if let (Some(old), Some(new)) = (old, new) {
+                    let old = get_changelog(old.root());
+                    let new = get_changelog(new.root());
+                    match (old, new) {
+                        (Ok(old), Ok(new)) => {
+                            let diff = changelog_diff(old, new);
+                            if diff != "" {
+                                println!("--> Additions to CHANGELOG\n{}", diff)
+                            }
+                        }
+                        _ => println!("--> Error while reading CHANGELOG"),
```

Comment:
I'm still a bit confused about the motivation for this kind of error handling. Let's talk OOB.